src/base-element.js

```diff
@@ -750,6 +750,25 @@ export class BaseElement {
     }
   }
 
+  /**
+   * Configures the supplied element to have an "intrinsic" layout. The
+   * exact interpretation of "intrinsic" depends on the element's layout.
+   *
+   * If `opt_replacedContent` is specified, it indicates whether the "replaced
+   * content" styling should be applied. Replaced content is not allowed to
+   * have its own paddings or border.
+   *
+   * @param {!Element} element
+   * @param {boolean=} opt_replacedContent
+   * @public @final
+   */
+  applyIntrinsicContent(element, opt_replacedContent) {
```

Comment:
Do we call this from anywhere? 
We have `applyFillContent` mostly because some components like carousel need to apply it at runtime. I don't see a need for this (yet).

Reply:
You're right I'll remove it